log-factory/sinks/log-to-file/src/main/java/io/deephaven/internal/log/LoggerFactoryFile.java

```diff
@@ -40,10 +41,10 @@ private static boolean showThreadName() {
                 .getProperty("io.deephaven.internal.log.LoggerFactoryFile.showThreadName", "true"));
     }
 
-    private static TimeZone timeZone() {
+    private static ZoneId zoneId() {
         final String timeZone =
                 System.getProperty("io.deephaven.internal.log.LoggerFactoryFile.timeZone");
-        return timeZone == null ? TimeZone.getDefault() : TimeZone.getTimeZone(timeZone);
+        return timeZone == null ? ZoneId.systemDefault() : ZoneId.of(timeZone, ZoneId.SHORT_IDS);
```

Comment:
How confident are we about this? Presumable it's not being set often/ever. Maybe in Core+?